tests/codeception/common/_support/FixtureHelper.php

```diff
@@ -58,6 +58,30 @@ public function globalFixtures()
     }
 
     /**
+     * Workaround used to load custom fixtures for the test
+     * @link https://github.com/Codeception/Codeception/issues/2258
+     * @link https://github.com/yiisoft/yii2/issues/5575
+     * @param array $fixtureConfig
+     */
+    public function loadFixturesForTest(array $fixtureConfig)
+    {
+        $fixtures = ArrayHelper::merge(
```

Comment:
I've removed redundant line in `tests/codeception/frontend/functional/fixtures/signup_cest.php`

>  i recommended is to add getFixture as public and make loadFixtures($fixtures) as public

That was my first idea to check too. But the test code became a mess and I can't make it work at all. This is because Codeception uses Reflection to run public methods in modules.

So if we make the `$this->loadFixtures()` public, the only way to use it from the test is `$I->loadFixtures()` and this approach makes `$this` points to `tests\codeception\frontend\functional\SignupCest` instance not the `tests/codeception/common/_support/FixtureHelper` one. So all code uses `$this` in `tests/codeception/common/_support/FixtureHelper` will be broken. (: